Holovibes/includes/API.hxx

```diff
@@ -546,6 +546,93 @@ inline void set_fft_shift_enabled(bool value)
     pipe_refresh();
 }
 
+/*!
+ * \name Artery Mask
+ *
+ */
+inline bool get_artery_mask_enabled() { return GET_SETTING(ArteryMaskEnabled); }
+inline void set_artery_mask_enabled(bool value)
+{
+    UPDATE_SETTING(ArteryMaskEnabled, value);
+    pipe_refresh();
+}
+
+/*!
+ * \name Vein Mask
+ *
+ */
+inline bool get_vein_mask_enabled() { return GET_SETTING(VeinMaskEnabled); }
+inline void set_vein_mask_enabled(bool value)
+{
+    UPDATE_SETTING(VeinMaskEnabled, value);
+    pipe_refresh();
+}
+
+/*!
+ * \brief Time Window
+ *
+ */
+
+inline int get_time_window() { return GET_SETTING(TimeWindow); }
+inline void set_time_window(int value)
+{
+    UPDATE_SETTING(TimeWindow, value);
+    pipe_refresh();
+}
+
+/*!
+ * \name Otsu
+ *
+ */
+inline bool get_otsu_enabled() { return GET_SETTING(OtsuEnabled); }
+inline void set_otsu_enabled(bool value)
+{
+    UPDATE_SETTING(OtsuEnabled, value);
+    pipe_refresh();
+}
+
+/*!
+ * \name Vesselness Sigma
+ *
+ */
+inline double get_vesselness_sigma() { return GET_SETTING(VesselnessSigma); }
+inline void set_vesselness_sigma(double value)
+{
+    UPDATE_SETTING(VesselnessSigma, value);
+    pipe_refresh();
+}
+inline int get_min_mask_area() { return GET_SETTING(MinMaskArea); }
+inline void set_min_mask_area(int value) { return UPDATE_SETTING(MinMaskArea, value); }
+
+inline OtsuKind get_otsu_kind() { return GET_SETTING(OtsuKind); }
+inline void set_otsu_kind(OtsuKind value) { return UPDATE_SETTING(OtsuKind, value); }
+inline int get_otsu_window_size() { return GET_SETTING(OtsuWindowSize); }
+inline void set_otsu_window_size(int value) { UPDATE_SETTING(OtsuWindowSize, value); }
+inline float get_otsu_local_threshold() { return GET_SETTING(OtsuLocalThreshold); }
+inline void set_otsu_local_threshold(float value) { UPDATE_SETTING(OtsuLocalThreshold, value); }
+
+/*!
+ * \name Bwareafilt
+ *
+ */
+inline bool get_bwareafilt_enabled() { return GET_SETTING(BwareafiltEnabled); }
+inline void set_bwareafilt_enabled(bool value)
+{
+    UPDATE_SETTING(BwareafiltEnabled, value);
+    pipe_refresh();
+}
+
+/*!
+ * \name Bwareafilt
+ *
+ */
+inline bool get_bwareaopen_enabled() { return GET_SETTING(BwareaopenEnabled); }
+inline void set_bwareaopen_enabled(bool value)
+{
+    UPDATE_SETTING(BwareaopenEnabled, value);
+    pipe_refresh();
+}
+
```

Comment:
paye attention with the api refacto and maybe add more comment and remove debug function